recipes/ede-php-autoload

```diff
@@ -0,0 +1,3 @@
+(ede-php-autoload :fetcher github
+                  :repo "stevenremot/ede-php-autoload"
+                  :files ("*.el" "ede-php-autoload"))
```

Comment:
Is it enough to use `:defaults` instead of `*.el` ?

See
- https://github.com/melpa/melpa#recipe-format